test/react-native/features/fixtures/app/scenario_js/app/scenarios/BreadcrumbsJsManualScenario.js

```diff
@@ -6,6 +6,10 @@ export class BreadcrumbsJsManualScenario extends Scenario {
     const metaData = {
       from: 'javascript'
     }
+
+    // ensure that circular references are safely handled
+    metaData.circle = metaData
```

Comment:
Can we add this to [the expected breadcrumb](https://github.com/bugsnag/bugsnag-js/blob/250045158bea921665d51c0383d60a50cbc78e9d/test/react-native/features/fixtures/expected_breadcrumbs/JsManualScenario.json)?